scripts/core/superdesk-api.d.ts

```diff
@@ -2891,7 +2891,8 @@ declare module 'superdesk-api' {
         };
         entities: {
             article: {
-                translate(article: IArticle, language: string): Promise<IArticle>;
+                createTranslation(article: IArticle, language: string): Promise<IArticle>;
```

Comment:
`translate` was a better name - it'd be good to go back to it. Since this is exposed to API - we should keep it single purpose and not trigger opening of an article. There's already a separate method for this in the API.

Regarding `translateAndPatch` - why is that needed at all? We already have a "translate" method. Patching should be available already via an independent method.